TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/osi_utils.py

```diff
@@ -57,6 +58,20 @@ def __init__(self, config: Dict) -> None:
             self.tags = convert_str_tags_to_dict(tags)
 
 
+def get_assume_role_session(role_arn, session_name) -> boto3.Session:
```

Comment:
I'd love to see an example of what the caller sees if this fails. It seems like one of the more likely to fail pieces, given the general complication of IAM stuff.